### DISCUSSION TITLE
Android won't work

### DISCUSSION THREAD

**Discussion by woelfle2004** [srcnalt/OpenAI-Unity#10] [Q&A] [answered]
Does anybody got this great package to work on Android?
On Windows it works like a charm.
Thanks all

**Comment by berkingokyolcu** [comment]
Yeah same problem for me too. I am guessing it happens because we put api key on users. I don't know if it's because of that. I don't know how to but we can try integrate it to script we may solve this problem.
If this solves your problem can you show me too?

**Reply by woelfle2004** [reply]
I am still working on it but sending a curl command doesn't seem to work as well

**Comment by srcnalt** [comment, marked as answer]
I created this PR (https://github.com/srcnalt/OpenAI-Unity/pull/12) for letting developers use API key while creating the OpenAIApi instance. But again, this is not a secure way to use it. Could you check and see if that works for you?

**Comment by berkingokyolcu** [comment]
@woelfle2004 Did it work? i dont know hardcode so if that solution worked ill try it.

**Reply by woelfle2004** [reply]
Yes it works well on Android
